crates/re_log_types/src/component_types/instance_key.rs

```diff
@@ -70,6 +70,12 @@ impl std::fmt::Display for InstanceKey {
     }
 }
 
+impl From<u64> for InstanceKey {
+    fn from(value: u64) -> Self {
```

Comment:
adding `#[inline]` to things like these can make a _huge_ difference when called from another crate!

```suggestion
    #[inline]
    fn from(value: u64) -> Self {
```